### DISCUSSION TITLE
Updating from v8 to latest

### DISCUSSION THREAD

**Discussion by vguna** [jomjol/AI-on-the-edge-device#2786] [General] [answered]
Hey guys,
I thought maybe it's a good idea to update from my existing v8 to latest version. I read the docs about the upgrade. I'm wondering though what's the quickest way if I don't care about the old config, as I would plan to setup everything from scratch anyway :). Can I then go straight to latest via OTA and all good? Are there any drawbacks going to a more recent version? I saw some crashes while playing with the UI on v8... most likely due to CPU getting overloaded. Were there improvements in that direction with the more recent versions?

**Comment by jomjol** [comment]
What do you mean by v8?  There is no version 8 in our systematic.

**Comment by vguna** [comment]
Hey, v8 like in

![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/3257311/811397c2-1621-45fd-afd6-09eaadc0e248)

**Comment by jomjol** [comment, marked as answer]
Okay - I forgott about this old naming system. Meanwhile almost everything changed. There the OTA will also not work.

Please make it completly from scratch: format SD-card, flash the firmware by hand and run through the setup.

Aftwards the OTA will work fine.

**Comment by vguna** [comment]
Alrighty, thanks!
